sale_timesheet_invoice_description/README.rst

```diff
@@ -14,6 +14,8 @@ Usage
 
 To use this module, you need to:
 
+#. Go to *Sales -> Configuration -> Settings -> Quotations & Sales* and set
```

Comment:
This paragraph should go in the Configuration section and include the word "Default".